CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v0.3.0](https://github.com/voxpupuli/puppet-vault_lookup/tree/v0.3.0) (2022-06-17)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-vault_lookup/compare/v0.2.0...v0.3.0)
+
+**Implemented enhancements:**
+
+- \(MODULES-11321\) Use new Puppet http runtime; require Puppet 6.11 or newer [\#50](https://github.com/voxpupuli/puppet-vault_lookup/pull/50) ([tvpartytonight](https://github.com/tvpartytonight))
```

Comment:
`metadata.json` says 6.16+ since https://github.com/voxpupuli/puppet-vault_lookup/commit/572bc4d22f79f6c1cbb0a3ff0867856a012fcf66 -- who's right? ;-)

Reply:
odd. I thought that got introduced in 6.10 :D . @tvpartytonight can you check that?

Reply:
I was going off the release notes [here]( https://puppet.com/docs/puppet/6/release_notes_puppet.html#release_notes_puppet_x-16-0).

Reply:
I updated the title, @bastelfreak  can you regenerate?

Reply:
regenerated